src/index.js

```diff
@@ -119,7 +114,10 @@ const initDataMap = container => {
     console.error("Cannot init maps without a key - set the data-spreadsheet-key attribute");
     return;
   }
-  initDom(container);
+  render(<App sheetKey={sheetKey} />, container);
+
+  return null;
```

Comment:
have been replacing everything after this - `initDom` is done but need to finish `initMap` `initTable` `initDetails`